imap-proto/src/parser/core.rs

```diff
@@ -1,6 +1,4 @@
-use nom::{self, IResult};
-
-use std::str;
```

Comment:
I don't think we've come to agreement on the removal of the `std::str` import?

Reply:
Yes. I forgot it. See last comment.